src/modules/interaction/ParticipantInteraction.tsx

```diff
@@ -1,147 +1,167 @@
-      },
-      {
-        id: 1,
-        name: 'Exchange 2',
-        description: 'Exchange 2 Description',
-        instructions: 'Instructions',
-        participantInstructionsOnComplete: ``,
-        cue: `Merci! À présent, pourriez-vous décrire s'il s'agissait plus de formes réelles ou imaginaires? Réalistes ou abstraites?`,
-        order: 2,
-        messages: [],
-        assistant: artificialAssistant,
-        triggers: [],
-        started: false,
-        completed: false,
-        dismissed: false,
-        softLimit: 5,
-        hardLimit: 0,
-        createdAt: new Date(),
-        updatedAt: new Date(),
-      },
-      {
-        id: 2,
-        name: 'Exchange 3',
-        description: 'Exchange 3 Description',
-        instructions: 'Instructions',
-        participantInstructionsOnComplete: ``,
-        cue: `Merci! Une dernière question s’il vous plaît! Où se trouvait votre corps par rapport à ces images?  Vous les observiez depuis un point de vue extérieur, depuis le bas ou le haut ou latéralement, ou alors aviez-vous la sensation d'être immergé dans un espace qui vous entoure, d'être transporté dans un lieu?`,
-        order: 3,
-        messages: [],
-        assistant: artificialAssistant,
-        triggers: [],
-        started: false,
-        completed: false,
-        dismissed: false,
-        softLimit: 5,
-        hardLimit: 0,
-        createdAt: new Date(),
-        updatedAt: new Date(),
-      },
-      {
-        id: 3,
-        name: 'Exchange 4',
-        description: 'Exchange 4 Description',
-        instructions: 'Instructions',
-        participantInstructionsOnComplete: ``,
-        cue: `Merci beaucoup pour vos réponses! Avez-vous quelque chose à ajouter?`,
-        order: 4,
-        messages: [],
-        assistant: artificialAssistant,
-        triggers: [],
-        started: false,
-        completed: false,
-        dismissed: false,
-        softLimit: 1,
-        hardLimit: 0,
-        createdAt: new Date(),
-        updatedAt: new Date(),
-      },
-    ],
+    participant: currentMember,
+    exchanges: { exchangesList: [] },
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  };
+
+  // Define a default exchange object using default settings
+  const defaultExchange: Exchange = {
+    ...defaultSettingsValues.exchanges.exchangesList[0],
+    messages: [],
+    assistant: defaultAssistant,
+    started: false,
+    completed: false,
+    dismissed: false,
     createdAt: new Date(),
     updatedAt: new Date(),
   };
 
-  function load(key: string): Interaction {
-    const item = window.sessionStorage.getItem(key);
-    return item != null ? JSON.parse(item) : defaultInteraction;
+  /**
+   * @function createTemplate
+   * @description Creates and returns a new `Interaction` object by merging default settings with chat and exchange settings.
+   * @returns {Interaction} A fully constructed `Interaction` object with merged settings.
+   */
+  function createTemplate(): Interaction {
```

Comment:
Isn't this more like a `createInteractionFromTemplate` function? Perhaps a better naming convention.